packages/headless/src/features/recommendation/recommendation-state.ts

```diff
@@ -10,9 +10,25 @@ export const getRecommendationInitialState = (): RecommendationState => ({
 });
 
 export interface RecommendationState {
+  /**
+   * Specifies the ID of the recommendation interface.
+   * @default Recommendation
+   */
   id: string;
+  /**
+   * The list of recommendations.
+   */
   recommendations: Result[];
+  /**
+   * The execute time for the recommendation request, in milliseconds.
```

Comment:
I would suggest `The time it took to complete the recommendation request, in milliseconds`.